stl/inc/xloctime

```diff
@@ -538,7 +538,8 @@ protected:
 
 private:
     ios_base::iostate __CLRCALL_OR_CDECL _Getint(_InIt& _First, _InIt& _Last, int _Lo, int _Hi, int& _Val,
-        const _Ctype& _Ctype_fac) const { // get integer in range [_Lo, _Hi] from [_First, _Last)
+        const _Ctype& _Ctype_fac,
+        const int _MaxLen = -1) const { // get integer in range [_Lo, _Hi] from [_First, _Last)

```

Comment:
I believe this will break binary compatibility because it's changing the signature of a separately compiled function. (Much iostreams code is separately compiled; a subtle indication is the explicit calling convention `__CLRCALL_OR_CDECL`.) `dumpbin /exports` and grepping for `_Getint` should reveal whether the signature is changing. (We have a build step to validate that the export surface is unchanged, but we still need to port it from the MSVC-internal MSBuild system to the GitHub CMake/Ninja build system - sorry!)

It may be possible to fix this while preserving bincompat (as @statementreply encountered and solved a similar problem). I'll have time after CppCon to investigate in detail although another maintainer may have time before then.

Reply:
Thank you for the details. Hopefully, in this case, it was easy to find another solution without touching the function signature thanks to `_Hi` parameter.